### PR TITLE
Added japenese translations for janus vulnerability

### PR DESCRIPTION
**Pull request by poseidontor**

### CODE REVIEW

File: locales/ja/117_android_janus_vulnerability/name.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Janusの脆弱性(CVE-2017-13156)
```
Comment (rajan262) [inline locales/ja/117_android_janus_vulnerability/name.md:2]: There are no newlines here.

File: locales/ja/117_android_janus_vulnerability/compliant.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+ v2以上の方式を使用してapkに署名してください。
```
Comment (rajan262) [inline locales/ja/117_android_janus_vulnerability/compliant.md:2]: Add newline at start and end.

File: locales/ja/117_android_janus_vulnerability/description.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+このアプリケーションは 署名方式v1 で署名されているため、パッチの適用されていない Android 5.1.1 から 8.0 において Janusの脆弱性があります。V1と、v2またはv3の2重署名されているアプリは、ロールバック保護を実施しないv2 以上をサポートする特定のパッチ未適用の Android バージョンのデバイスでは、依然として脆弱性があることに注意してください。
```
Comment (rajan262) [inline locales/ja/117_android_janus_vulnerability/description.md:2]: Add newline at start and end.

File: locales/ja/117_android_janus_vulnerability/non_compliant.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
+アプリケーションはv1方式で署名されています。
```
Comment (rajan262) [inline locales/ja/117_android_janus_vulnerability/non_compliant.md:2]: Add newline at start and end.

File: locales/ja/117_android_janus_vulnerability/question.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
+アプリケーションは署名方式v1で署名されていますか?
```
Comment (rajan262) [inline locales/ja/117_android_janus_vulnerability/question.md:2]: There are no newlines here.

File: locales/ja/117_android_janus_vulnerability/success_message.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+
+アプリケーションは署名方式v2以上で署名されており、Janusの脆弱性に対して脆弱ではありません。
```
Comment (rajan262) [inline locales/ja/117_android_janus_vulnerability/success_message.md:2]: There are no newlines here.

**Review by rajan262 — APPROVED** [review]
Looks good

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**